pages/common/git-config.md

```diff
@@ -28,10 +28,10 @@
 
 `git config --global --unset alias.unstage`
 
-- Edit the Git configuration for the current repository in the default editor:
+- Edit the local Git configuration (`.git/config`) in the default editor:
 
 `git config --edit`
 
-- Edit the global Git configuration in the default editor:
+- Edit the global Git configuration (`~/.gitconfig` by default or `$XDG_CONFIG_HOME/git/config` if such a file exists) in the default editor:
```

Comment:
Not sure whether we should mention the path. Is it really the same among the supported OSes?